base_classes/NXtransformations.nxdl.xml

```diff
@@ -108,6 +126,143 @@
 		* depends_on
 
 		as needed.
+
+		**Example 1: goniometer**
+
+		Position a sample mounted on a goniometer in the :ref:`McSTAS coordinate frame&lt;Design-CoordinateSystem&gt;`.
+
+		The sample is oriented as follows
+
+		.. math:: X_\text{lab} = R(\vec{v}_\omega, \omega) . 
+								 T(\vec{v}_z, \text{sam}_z) . 
+								 T(\vec{v}_y, \text{sam}_y) . 
+								 T(\vec{v}_x, \text{sam}_x) . 
+								 R(\vec{v}_\chi, \chi) . 
+								 R(\vec{v}_\varphi, \varphi) . X_s
```

Comment:
there seems to be a issue here, \varphi is not \phi or should \varphi be \phi, cuz the code block below should match that example

Also X_s and X_lab is ambiguous in the description we talk also about McStas so in this case we would have already three reference frames, I think at least one should explicitly state that lab and McStas are equivalent